core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2603,4 +2603,13 @@ package object config {
       .stringConf
       .toSequence
       .createWithDefault("org.apache.spark.sql.connect.client" :: Nil)
+
+  private[spark] val ABORT_STAGE_AFTER_CANCEL_TASKS =
+    ConfigBuilder("spark.scheduler.stage.abortStageAfterCancelTasks")
```

Comment:
is this kind of a legacy config to restore a legacy behavior?

Reply:
Right, should probably add the "legacy" key word.